simulator_launch/launch/simulator.launch.xml

```diff
@@ -24,6 +25,47 @@
     </include>
   </group>
 
+  <!-- perception module -->
+  <group>
+    <push-ros-namespace namespace="perception"/>
+    <!-- object recognition -->
+    <group if="$(var perception/enable_object_recognition)">
+      <push-ros-namespace namespace="object_recognition"/>
+      <!-- detection module -->
```

Comment:
Could you remove this?

Reply:
https://github.com/tier4/autoware_launch/pull/227/commits/9408573517f02bb5c03fbf730033d314a3282478
DONE